stack.yaml

```diff
@@ -1,16 +1,29 @@
-resolver: nightly-2022-06-20
+# resolver: nightly-2022-06-06
+# resolver: nightly-2022-06-20
+resolver: nightly-2022-08-04
+# resolver: lts-19.17
+
 packages:
 - cachix-api
 - cachix
 extra-deps:
-- hercules-ci-cnix-store-0.3.3.0
+- hercules-ci-cnix-store-0.3.3.1
 - netrc-0.2.0.0
 - nix-narinfo-0.1.0.1
 - protolude-0.3.2
 - servant-auth-swagger-0.2.10.1
 - cabal-pkg-config-version-hook-0.1.0.0
+# - aeson-2.1.0.0
+# - generically-0.1
 # https://github.com/haskell-hvr/netrc/issues/5
 allow-newer: true
 nix:
-  packages: []
+  enable: true
+  # packages: [lzma zlib nix boost haskell-language-server]
   shell-file: ./nix/stack-shell.nix
+
+# https://github.com/alphaHeavy/lzma-conduit/issues/17#issuecomment-357370390
+# https://stackoverflow.com/a/63751678
+
+ghc-options:
+  cachix: -fobject-code -lnixstore
```

Comment:
Hmm, what is the error you get?

Reply:
It was `commitAndReleaseBuffer: invalid argument (invalid character)`